src/briefcase/commands/new.py

```diff
@@ -12,9 +12,10 @@
     is_valid_bundle_identifier,
     make_class_name,
 )
+from briefcase.exceptions import TemplateUnsupportedVersion
 from briefcase.integrations import git
 
-from .base import BaseCommand, BriefcaseCommandError, TemplateUnsupportedVersion
+from .base import BaseCommand, BriefcaseCommandError
```

Comment:
`BriefcaseCommandError` should be imported from it's original source (`briefcase.exceptions`)